pom.xml

```diff
@@ -27,6 +27,32 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-log4j2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.22.0</version>
```

Comment:
extract this version as a property. We are using it in 2 places it'd be good to have ti as a property in case we change it in one place and forget about it in the other